packages/aft/lib/src/commands/generate/generate_workflows_command.dart

```diff
@@ -24,12 +24,27 @@ class GenerateWorkflowsCommand extends AmplifyCommand {
           !falsePositiveExamples.contains(package.name)) {
         continue;
       }
-      // Some packages contain only native code/dependencies and do not need
-      // to be tested by Dart/Flutter analyze or test.
+      // Some packages do not need to be tested.
```

Comment:
Instead of shortening it, maybe we need to add more context.

```suggestion
      // Packages without a `lib/` folder generally contain only native code,
      // e.g. `amplify_flutter_android`. These packages are tested via their
      // parent package, e.g. `amplify_flutter`, and do not require a workflow
      // of their own. 
```

Reply:
good suggestion, thanks